server/constants/feature.ts

```diff
@@ -1,11 +1,34 @@
 enum FEATURE {
+  // Collective page main sections (1-5)
+  // 1. Contribute
+  CONTRIBUTE = 'CONTRIBUTE',
+  CONTRIBUTIONS = 'CONTRIBUTIONS', // show contributions the collective has made
+  RECURRING_CONTRIBUTIONS = 'RECURRING_CONTRIBUTIONS',
+  // 2. Events/Projects - no features/subsections at the moment
+  EVENTS = 'EVENTS',
+  PROJECTS = 'PROJECTS',
+  // 3. Budget
+  BUDGET = 'BUDGET',
+  TRANSACTIONS = 'TRANSACTIONS',
+  USE_EXPENSES = 'USE_EXPENSES',
+  COLLECTIVE_GOALS = 'COLLECTIVE_GOALS',
+  TOP_FINANCIAL_CONTRIBUTORS = 'TOP_FINANCIAL_CONTRIBUTORS',
+  ALL_FINANCIAL_CONTRIBUTORS = 'ALL_FINANCIAL_CONTRIBUTORS',
+  // 4. Connect
+  CONNECT = 'CONNECT',
```

Comment:
Sounds to me that `CONNECT` is not a feature, it's a section from the collective page.